web/src/labs/highlighter/index.ts

```diff
@@ -0,0 +1,28 @@
+const escapeRegExp = (str: string): string => {
+  return str.replace(/[.*+?^${}()|[\]\\]/g, "\\$&");
+};
+
+function dfsAndHighlight(node: HTMLElement, keyword: string) {
```

Comment:
The function name is a bit non-standard, how about `walkthroughNodeWithKeyword`?